### PR TITLE
Adding an example for overscroll-behavior-x

### PR DESCRIPTION
**Pull request by doanaminh**
### Description
Added another dimension to the existing overscroll-behavior example.
Isolated only the x-axis to emphasize that the overscroll-behavior property should only work in one dimension.

### Motivation
Created an example with 2 dimensions to visually emphasize the effects of overscroll-behavior-x.

### Additional details
There was not an existing example.

### CODE REVIEW

File: live-examples/css-examples/basic-box-model/overscroll-behavior-x.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="overscroll-behavior">
```
Comment (NiedziolkaMichal) [inline live-examples/css-examples/basic-box-model/overscroll-behavior-x.html]: Attribute `data-property` should have a value `overscroll-behavior-x`.

File: live-examples/css-examples/basic-box-model/overscroll-behavior-x.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    <p>If you have <code class="language-css">overscroll-behavior-x: 
+                        auto;</code> selected the outer container 
+                        will start to scroll.</p>
+                    If you have <code class="language-css">overscroll-behavior-x: 
+                        contain;</code> selected, the outer container will not scroll unless you move your cursor out of the inner container and try to perform scroll on the outer container. 
+
+                        **Note that this applies only to the x-axis. You should still be able to scroll past in the y-axis.
+                </div>
+                <div id="element-right">
+                    This is sample text to fill up space. Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius veniam, animi laudantium quibusdam vitae cumque officiis in eos laborum molestiae quasi ex eaque minima architecto, inventore natus fuga tempora accusamus velit deserunt numquam, illo nemo labore accusantium. Earum nihil placeat sequi veritatis tempora, corrupti et necessitatibus quos! Rem, sed quod!
```
Comment (NiedziolkaMichal) [inline live-examples/css-examples/basic-box-model/overscroll-behavior-x.html]: This paragraph is way too long, it takes forever to scroll through it on the phone. Also visible content of the box should indicate that there is some additional content on the right. On many devices, scrollbar is not shown until scrolling through the content, so users can easily think that there is more text only at the bottom:
image

File: live-examples/css-examples/basic-box-model/overscroll-behavior-x.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                </div>
+            </div>
+            <div id="last-element">
+                This is just some sample text. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eum at aut eius aliquam blanditiis itaque unde reiciendis? Maxime obcaecati possimus dicta quidem nemo consectetur facere officiis assumenda laudantium. Totam vitae nisi pariatur voluptate et architecto officia ullam cum recusandae! Earum, soluta eveniet corrupti quibusdam libero numquam! Incidunt quidem libero, et ratione aspernatur, officia quisquam cum maxime consectetur pariatur tenetur sint inventore doloremque beatae quibusdam ipsam harum minima nulla deserunt ex illum? Obcaecati animi ab reprehenderit esse adipisci, optio vitae incidunt. Reprehenderit corrupti, autem eum laboriosam in cum! Reprehenderit repellendus, eos magnam deleniti, excepturi sequi rem sint fugit corrupti autem quaerat.
```
Comment (NiedziolkaMichal) [inline live-examples/css-examples/basic-box-model/overscroll-behavior-x.html]: I think most of this text is unnecessary. It's good to allow users to scroll through the content horizontally and vertically, so they can see how CSS property affects both axes, but a single short sentence would be enough to achieve this.

**Review by NiedziolkaMichal — CHANGES_REQUESTED** [review]
Thanks a lot for this example, it looks good. I have added some comments about how I think it could be improved.

**Review by doanaminh — COMMENTED** [review]
Made changes as requested by @sideshowbarker.

**Review by doanaminh — COMMENTED** [review]
Completed changes required to pass checks.

**Review by doanaminh — COMMENTED** [review]
3rd attempt at getting changes fixed and rebasing the branch to get ready to merge.

**Comment by NiedziolkaMichal** [comment]
It looks good :) Just fix those two eslint errors and we can merge:
```
live-examples/css-examples/basic-box-model/overscroll-behavior-x.css
 46:2  ✖  Delete "·"                            prettier/prettier
 46:2  ✖  Unexpected whitespace at end of line  no-eol-whitespace
```

**Comment by doanaminh** [comment]
Okay, I deleted the whitespace so hopefully, it can merge this time!

**Review by NiedziolkaMichal — APPROVED** [review]

**Comment by NiedziolkaMichal** [comment]
Thank you for your contribution, everything looks good.

**Pull request merged**